res/css/views/right_panel/_BaseCard.pcss

```diff
@@ -219,6 +219,10 @@ limitations under the License.
     }
 }
 
+.mx_UserInfo {
+    font-size: var(--cpd-font-size-body-md);
```

Comment:
Inferring from the title of this PR, did you mean to set the size to `body-sm` here?

Reply:
You're totally right! thank you